src/EditorFeatures/Core.Wpf/InlineDiagnostics/InlineDiagnosticsTaggerProvider.cs

```diff
@@ -57,12 +57,6 @@ internal sealed class InlineDiagnosticsTaggerProvider : AbstractDiagnosticsAdorn
             _classificationTypeRegistryService = classificationTypeRegistryService;
         }
 
-        protected sealed override bool SupportsDiagnosticMode(DiagnosticMode mode)
-        {
-            // We support inline diagnostics in both push and pull (since lsp doesn't support inline diagnostics yet).
-            return true;
-        }
```

Comment:
no need for thsi switch anymore.  since we're always in pull diags now, and inline diagnostics is always supported there.